<launch>
    <node name="erratic_manager" pkg="wubble_controllers" type="erratic_player" output="screen" required="true">
        <param name="port_name" type="str" value="/dev/erratic"/>
        <param name="odometry_topic" type="str" value="odom"/>
        <param name="battery_topic" type="str" value="battery_state"/>
        <param name="odometry_frame_id" type="str" value="odom"/>
    </node>
    
    <node name="ax12_manager" pkg="wubble_controllers" type="manager.py" output="screen" required="true">
        <param name="port_name" type="str" value="/dev/ax12"/>
        <param name="baud_rate" type="int" value="1000000"/>
        <param name="max_motor_id" type="int" value="25"/>
        <param name="update_rate" type="int" value="1"/>
    </node>
    
    <node name="laser_tilt_spawner" pkg="wubble_controllers" type="driver_ctrl.py" args="start $(find wubble_controllers)/src/wubble_controllers/laser_tilt_ax12.py" output="screen"/>
    <node name="camera_pan_tilt_spawner" pkg="wubble_controllers" type="driver_ctrl.py" args="start $(find wubble_controllers)/src/wubble_controllers/camera_pan_tilt_ax12.py" output="screen"/>
    <node name="arm_spawner" pkg="wubble_controllers" type="driver_ctrl.py" args="start $(find wubble_controllers)/src/wubble_controllers/arm_ax12.py" output="screen"/>
</launch>
